classifiers/__init__.py

```diff
@@ -61,6 +62,8 @@
     workday_classifier,
     deberta_review_classifier, 
     bert_sentiment_german,
+    tiktoken_length_classifier,
+    chunked_sentence_complexity,
     special_character_classifier,
     chunked_sentence_complexity
```

Comment:
`chunked_sentence_complexity` is duplicated

Reply:
Removed duplicated entry.